src/reducers/RootReducer.ts

```diff
@@ -0,0 +1,8 @@
+import {combineReducers} from 'redux';
+
+import {loginReducer} from './loginReducer';
+import {feedsReducer} from './feedsReducer';
+
+export const RootReducer = combineReducers({loginReducer, feedsReducer});
+
+export type AppState = ReturnType<typeof RootReducer>;
```

Comment:
AppState interface -> combination of reducers state
interface AppState extends LoginState, ... {}